xbmc/video/ContextMenus.cpp

```diff
@@ -205,7 +221,7 @@ bool CVideoResume::Execute(const std::shared_ptr<CFileItem>& itemIn) const
 #endif
 
   item.SetStartOffset(STARTOFFSET_RESUME);
-  SetPathAndPlay(item);
+  SetPathAndPlay(item, "");
```

Comment:
This looks ugly. I know default params are bad in some cases but isn't it better to keep an empty string as default argument?
Another alternative would be std:optional and pass nullopt here (not really an advantage but would at least convey the idea of something "optional" better than an empty string).

Reply:
I'd say, lets not over-engineer this. This is a source file local helper function only.

Reply:
So why not a default value?

Reply:
Ideally by looking at `SetPathAndPlay(item, "");` doesn't ring any bell without looking at the implementation or documentation (in this case 2 levels since it's only mentioned in video_utils...).

It's not over-engineering it (well the std::optional definitely is) it's just trying to make it read better without having to inspect methods to understand what's being done.

Reply:
I hate default prams! And please don't try to ever to force me to introduce one. :-) 
You have seen the regression I introduced with external players not working anymore, just because my IDE auto-completed without filling the optional param, haven't you.